.travis.yml

```diff
@@ -26,30 +26,30 @@ cache:
     - $HOME/.composer/cache/files
 
 env:
-  matrix: SYMFONY_VERSION=3.4
+  matrix: SYMFONY_VERSION=^3.4@dev
   global:
-    - SYMFONY_DEPRECATIONS_HELPER=48
-    - SYMFONY_PHPUNIT_DIR=.phpunit SYMFONY_PHPUNIT_REMOVE="symfony/yaml" SYMFONY_PHPUNIT_VERSION=5.7
+    - SYMFONY_DEPRECATIONS_HELPER=0
+    - SYMFONY_PHPUNIT_DIR=.phpunit SYMFONY_PHPUNIT_REMOVE="symfony/yaml"
+    - KERNEL_CLASS=Symfony\Cmf\Bundle\CoreBundle\Tests\Fixtures\App\Kernel
+    - SYMFONY_PHPUNIT_VERSION=5.7
     - TEST_INSTALLATION=false
 
 matrix:
   include:
     - php: 7.1
-      env: SYMFONY_VERSION=3.4.*
+      env: DEPS=dev SYMFONY_VERSION=^4@dev
```

Comment:
lets at least not do the 4.0 thing in this branch as well, there are too many things going on at once. (and this will fail because composer.json)

Reply:
This one is in allowed failures as i know that it will fail, but i whant to collect reasons for failings with 4.0 to fix them first.